tests/utils/index.js

```diff
@@ -1078,7 +1078,7 @@ const waitForDockerLogs = (container, ...regex) => {
       console.log('Found logs', logs, 'watched for', ...regex);
       reject(new Error('Timed out looking for details in logs.'));
       killSpawnedProcess(proc);
-    }, 10000);
+    }, 12000);
```

Comment:
I'm assuming this will get slightly slower over time as we add more stuff to generate (eg: if we added translations).

```suggestion
    }, 20000);
```